ansible/services/nix-komodo/.env

```
@@ -0,0 +1,130 @@
+
+## These compose variables can be used with all Komodo deployment options.
+## Pass these variables to the compose up command using `--env-file komodo/compose.env`.
+## Additionally, they are passed to both Komodo Core and Komodo Periphery with `env_file: ./compose.env`,
+## so you can pass any additional environment variables to Core / Periphery directly in this file as well.
+
+## Stick to a specific version, or use `latest`
+COMPOSE_KOMODO_IMAGE_TAG=latest
+
+## Note: ðŸš¨ Podman does NOT support local logging driver ðŸš¨. See Podman options here:
+## `https://docs.podman.io/en/v4.6.1/markdown/podman-run.1.html#log-driver-driver`
+COMPOSE_LOGGING_DRIVER=local # Enable log rotation with the local driver.
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)